BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -1258,8 +1258,9 @@ make_icosahedron(
  * If `triangulated` is `true`, the diagonal of each cell is oriented from (0,0) to (1,1)
  * in the cell coordinates.
  *
- * \tparam CoordinateFunctor that takes two `boost::graph_traits<Graph>::%vertices_size_type`
+ * \tparam CoordinateFunctor that takes three `boost::graph_traits<Graph>::%vertices_size_type`
```

Comment:
a function object providing `Point_3 operator()(size_type I, size_type J, size_type)` with `Point_3` being ....
Also backquote the entire interval not just `i` and `j`.